store/storetest/user_store.go

```diff
@@ -3368,11 +3368,11 @@ func testUserStoreGetProfilesNotInTeam(t *testing.T, ss store.Store) {
 	})
 
 	t.Run("get not in team 1, offset 0, limit 100000 after update", func(t *testing.T) {
-		result := <-ss.User().GetProfilesNotInTeam(teamId, false, 0, 100000, nil)
-		require.Nil(t, result.Err)
+		result, e := ss.User().GetProfilesNotInTeam(teamId, false, 0, 100000, nil)
```

Comment:
We prefer to use `err` instead of just `e` and probably you can use `users` instead of `result` because it is no longer a `store.StoreResult`. (This applies to the other cases in this file too).

Reply:
Good idea.  I used e, because the style checker was grumpy about me shadowing `err` defined on line 3278. Since that first err, is long-lived and reused, it's a tad tricky to meaningfully rename or scope it.  How about I opt to prefix the offending `err` assignments like so `userErr`?

Reply:
that works for me :)